src/status_im2/contexts/quo_preview/loaders/skeleton_list.cljs

```diff
@@ -17,19 +19,41 @@
                :value "Messages"}]}
    {:label "Blur?"
     :key   :blur?
+    :type  :boolean}
+   {:label "Animated?"
+    :key   :animated?
     :type  :boolean}])
 
 (defn cool-preview
   []
   (let [state (reagent/atom {:content       :messages
                              :blur?         false
+                             :animated?     true
                              :parent-height 600})]
     (fn []
       [rn/touchable-without-feedback {:on-press rn/dismiss-keyboard!}
        [rn/view {:padding-bottom 150}
+        [rn/view
+         {:style {:margin-vertical 40
+                  :padding-left    40
+                  :flex-direction  :row
+                  :align-items     :center}}
+         [text/text
+          {:size   :heading-1
+           :weight :semi-bold}
+          "Skeleton list"]
+         [rn/view
+          {:style {:width            20
+                   :height           20
+                   :border-radius    60
+                   :background-color colors/success-50
+                   :align-items      :center
+                   :justify-content  :center
+                   :margin-left      8}}
+          [icon/icon :i/check {:color colors/white :size 16}]]]
```

Comment:
Do we need this portion here? I think only two components have this green check with title, `toast` and `account_card`, don't sure if we need this.
@ilmotta Please can you take a look.

Reply:
IMO we do because is the component header. But I wait for @ilmotta

Reply:
if this is 
<img width="105" alt="Screenshot 2023-08-24 at 11 17 25" src="https://github.com/status-im/status-mobile/assets/22799766/3fb427cd-f6de-4d1e-97ee-2a2942df57c0">
then we do not need it in the preview screens. The green check mark is a way to say that designs are ready for development 👍

Reply:
Exactly as @J-Son89 said :)

Reply:
Thanks, guys! Im removing it